### PR TITLE
enh(MA-extract): move extract desc up & use for gen schema

### LINKED ISSUE #5439: [Multi Actions] Extract action configuration

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

fixes https://github.com/dust-tt/dust/issues/5439

### in multi actions mode

Screenshot 2024-06-06 at 15 00 48

### in legacy mode

Screenshot 2024-06-06 at 15 01 31

## Risk

Changes code used in legacy mode so could technically impact real people

## Deploy Plan

N/A

### CODE REVIEW

**Review by PopDaph — APPROVED** [review]
🙏🏻

**Review by spolu — COMMENTED** [review]
Have you tried changing the button text to 

`Generate from Instructions and Action Description` ?

Might be way too long....

File: front/components/assistant_builder/actions/ProcessAction.tsx [hunk 1]
```diff
@@ -316,11 +331,17 @@ export function ActionProcess({
 
   const generateSchemaFromInstructions = async () => {
     setEdited(true);
-
+    let fullInstructions = `${instructions}`;
+    if (description) {
+      fullInstructions += `\n${description}`;
```
[the remark this answers is not in the document]
Comment (fontanierh) [reply front/components/assistant_builder/actions/ProcessAction.tsx:336]: Yes that's a good idea

**Comment by fontanierh** [comment]
> Have you tried changing the button text to
> 
> Generate from Instructions and Action Description ?
> 
> Might be way too long....

@spolu  I tried and it felt super long... not sure if it's better

**Comment by spolu** [comment]
ack

**Pull request merged**